corehq/apps/hqwebapp/session_details_endpoint/tests.py

```diff
@@ -48,11 +49,11 @@ def test_session_details_view(self):
         self.assertEqual(200, response.status_code)
         self.assertJSONEqual(response.content, self.expected_response)
 
-    @override_settings(FORMPLAYER_INTERNAL_AUTH_KEY='123abc', DEBUG=False)
+    @override_settings(FORMPLAYER_INTERNAL_AUTH_KEY=b'123abc', DEBUG=False)
     def test_with_hmac_signing(self):
         assert not settings.DEBUG
         data = json.dumps({'sessionId': self.session_key, 'domain': 'domain'})
-        header_value = base64.b64encode(hmac.new('123abc', data, hashlib.sha256).digest())
+        header_value = base64.b64encode(hmac.new(b'123abc', data, hashlib.sha256).digest())
```

Comment:
`header_value` is a bytestring. `'bad signature'` on line 75 (GitHub won't let me add a comment there) should be a bytestring too.

Reply:
I got ```test_with_hmac_signing_fail``` to pass locally just by setting ```FORMPLAYER_INTERNAL_AUTH_KEY``` to a bytestring, without changing ```'bad signature'```.  Do you think this will fail in python 3?

Reply:
if `FORMPLAYER_INTERNAL_AUTH_KEY` needs to be bytes then we should add a comment about that in `settings.py` where it is set and update all envs in commcare-cloud (eventually). Looks like the value should also be changed to `b"secretkey"` in [dev_settings.py](https://github.com/dimagi/commcare-hq/blob/23860b99961e95d500ee00f446b788a7e2eeba84/dev_settings.py#L117).

Alternately, it may be easier to have the setting value be text (unicode) and convert it to bytes (`.encode('utf8')`) where necessary.

Reply:
Added 12d8ef30be592902b8573c1e2e76cf6924fe0126 to allow auth settings to be unicode